source/ranges.tex

```diff
@@ -8018,8 +8018,8 @@
 \begin{example}
 \begin{codeblock}
 string str{"the quick brown fox"};
-for (string_view word : views::split(str, ' ')) {
-  cout << word << '*';
+for (const auto& word : views::split(str, ' ')) {
```

Comment:
It should be simpler to use `auto`.

Reply:
Hi,
Sorry, what do you mean? I'm using `auto`. Do you mean avoiding the string_view conversion entirely on the next line?

Reply:
They mean just `auto`, not `const& auto`.

Reply:
Oh, alright, makes sense. Will amend.